type inputs

Krovatkin opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #67425
* __->__ #67424
* #67423

Differential Revision: [D31998565](https://our.internmc.facebook.com/intern/diff/D31998565)

Krovatkin committed:
type inputs

[ghstack-poisoned]

navahgar reviewed (approved):
Thanks a lot for fixing this quickly.

Krovatkin committed:
Update on "type inputs"


Differential Revision: [D31998565](https://our.internmc.facebook.com/intern/diff/D31998565)

[ghstack-poisoned]

navahgar commented:
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31998565).
